### PR TITLE
Hw 08

### PR DESCRIPTION
**Pull request by Ivan-paik**

### CODE REVIEW

File: HW_08/task3.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+lst = ["qwe", "wEr", "erT", "rTY", "YuI"]
+
+# v1. via definition
+def in_upper(symbol):
+    return(symbol.upper())
```
Comment (acman) [inline HW_08/task3.py]: nit: brackets can be omitted 
```suggestion
    return symbol.upper()
```

File: HW_08/task4.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# str, int, float, hex data
+lst = ["abc", "44", "0", 3, 1.5, .2, "2.2", "44", 2e1, 0x11, "4*7"]
+
+def digit_only(symbol):
+    if type(symbol) == int or type(symbol) == float:
```
Comment (acman) [inline HW_08/task4.py]: nit: nice function for checking type `isinstance()`

**Review by acman — APPROVED** [review]
LGTM!

File: HW_08/task4.py (hunk 2 above)
Comment (Ivan-paik) [reply HW_08/task4.py]: Changed the verification:
if isinstance(symbol, (float, int)): 

Made change in branche and will make new pool request

**Pull request merged**